forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -819,7 +819,19 @@ public static void addTriggerAbility(final KeywordInterface inst, final Card car
             inst.addTrigger(trigger);
         } else if (keyword.equals("Ascend")) {
             // Ascend trigger only for Permanent
-            if (card.isPermanent()) {
+        	if (card.isPlane()) { // Ascend trigger for The Golden City of Orazca
```

Comment:
Good catch

Imo for maintainability you can just combine it more since you'll only need to inject the different TriggerZones there